### PR TITLE
Document GHA release process

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Update documentation on release process
- Remove unused tooling

#### Rationale:

- Docs should be up to date

### CODE REVIEW

**Comment by yuumasato** [comment]
The `release_tools` directory is kinda empty, where should the `release_tools/README.md` be moved and renamed to?

**Comment by Mab879** [comment]
We could add section to the Developer Guide at https://complianceascode.readthedocs.io/en/latest/ for the release docs.

File: docs/index.rst [hunk 1]
```diff
@@ -50,7 +50,7 @@ ComplianceAsCode/content
     :glob:
     :caption: Release Tools Documentation
 
-    release_tools/README
+    docs/release_process
```
Comment (ggbecker) [inline docs/index.rst:53]:
```suggestion
    release_process
```

the context is already `docs` directory

**Review by ggbecker — CHANGES_REQUESTED** [review]
Since `release_tools/README.md` is being removed, you need to delete it from here:

https://github.com/ComplianceAsCode/content/blob/33949fb588c22da73c82bce4ddd75798a3568d60/docs/conf.py#L19-L21

File: docs/release_process.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+How to Perform An Upstream Release (GitHub)
+===========================================
+
+Stabilization Phase
+-------------------
+
+Before each release we iniate a stabilization period which is typically two weeks long.
+A stabilizaiton branch is branched from master and more extensive testing is done on the content.
+The goal is to have space and time to find and fix issues before the release.
+Everybody is welcome to test the stabilization branch and propose fixes.
```
Comment (ggbecker) [inline docs/release_process.md:10]:
```suggestion
Before each release we initiate a stabilization period which is typically two weeks long.
A stabilization branch is branched from master and more extensive testing is done on the content.
The goal is to have space and time to find and fix issues before the release.
Everyone is welcome to test the stabilization branch and propose fixes.
```

File: docs/release_process.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+- Create a new branch called **stabilization-vX.Y.Z**, e.g.: stabilization-v0.1.61
+
+- Create a new milestone for the next version
+
+- Update the milestone of open pull requests to the new one, if they have any
+
+- Update the milestone of open issues to the new one, if they have any
+
+- Close the milestone for the current release.
+  This makes the milestone less visible and reduces chance of PRs and Issues being added to it
```
Comment (ggbecker) [inline docs/release_process.md:26]:
```suggestion
  This makes the milestone less visible and reduces the chance of PRs and Issues being added to it
```

File: docs/release_process.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  accidentally.
+  It is still possible to add itens to the closed Milestone, just select it in the `closed` tab.
```
Comment (ggbecker) [inline docs/release_process.md:28]:
```suggestion
  It is still possible to add items to the closed Milestone, just select it in the `closed` tab.
```

File: docs/release_process.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+- Bump the version of `CMakeLists.txt` on the **master** branch.
+
+- Announce start of stabilization period on the mailing list
```
Comment (ggbecker) [inline docs/release_process.md:32]:
```suggestion
- Announce the start of stabilization period on the mailing list
```

File: docs/release_process.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+Everything necessary for the release is built by the release GitHub Action,
+which is triggered when a tag **v\*.\*.\*** is pushed to the repository. This tag
+should point to the **stabilization** branch that is ready to be released.
+
+This action will also create a release draft with release notes automatically
+generated. The way in which the action will categorize and break down changes is
+controlled by the `.github/workflows/release-changelog.json` configuration file.
+
+General rule is that the PR Titles will compose the body of the changelog.
```
Comment (ggbecker) [inline docs/release_process.md:70]:
```suggestion
The general rule is that the PR Titles will compose the body of the changelog.
```

File: docs/release_process.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+
+
+- Create and push **vX.Y.Z** tag to the GitHub repo. Wait for the release action
```
Comment (ggbecker) [inline docs/release_process.md:73]:
```suggestion
- Create and push a **vX.Y.Z** tag to the GitHub repo. Wait for the release action
```

**Review by ggbecker — CHANGES_REQUESTED** [review]

**Pull request merged**